### PR TITLE
[workers] Update changelog.md

### PR DESCRIPTION
**Pull request by kristianfreeman**

### CODE REVIEW

File: products/workers/src/content/platform/changelog.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
 
+## 12/22/2021
+
+- Async iteration (using `for` and `await`) on instances of `ReadableStream` is now available.
+
+## 12/10/2021
+
+- Raised the max value size in Durable Object storage from 32 KiB to 128 KiB.
+- `AbortSignal.timeout(delay)` returns an `AbortSignal` that will be triggered after the given number of milliseconds.
+- Preview implementations of the new `ReadableStream` and new `WritableStream` constructors are available behind the `streams_enable_constructors` feature flag.
```
Comment (kristianfreeman) [inline products/workers/src/content/platform/changelog.md:11]: @jasnell Is there any particular spot you think we should link here for documentation or announcements about this?

File: products/workers/src/content/platform/changelog.md [hunk 2]
```diff
@@ -1,5 +1,18 @@
...
+- `crypto.DigestStream` is a non-standard extension to the crypto API that supports generating a hash digest from streaming data. The `DigestStream` itself is a `WritableStream` that does not retain the data written into it; instead, it generates a digest hash automatically when the flow of data has ended. The same hash algorithms supported by `crypto.subtle.digest()` are supported by the `crypto.DigestStream`.
+- The `scheduler.wait()` API provides an `await`-able alternative to `setTimeout()`.
```
Comment (kristianfreeman) [inline products/workers/src/content/platform/changelog.md]: This is neat! @cloudflare/developer-advocacy anyone want to document this?

Comment (lukeed) [reply products/workers/src/content/platform/changelog.md]:
```suggestion
- Added early support for the `scheduler.wait()` API, which is [going through the WICG standardization process](https://github.com/WICG/scheduling-apis), to provide an `await`-able alternative to `setTimeout()`.
```

**Review by jkup — APPROVED** [review]
Love it! 👍🏻 with Luke's edit!

**Review by deadlypants1973 — APPROVED** [review]

**Pull request merged**